lib/config/virtual_container_host.go

```diff
@@ -155,6 +155,8 @@ type Connection struct {
 	ExtensionName string `vic:"0.1" scope:"read-only" key:"extension_name"`
 	// Whether the session connection is secure
 	Insecure bool `vic:"0.1" scope:"read-only" key:"insecure"`
+	// TargetThumbprint is the SHA-1 digest of the Target's public certificate
```

Comment:
That's what vCenter/ESX uses (see SDK related comments in the govmomi change).  I suppose we could support using either for the target host, but vCenter SDK will still be giving us SHA-1 to verify the ESX hosts.